DEVELOPING.md

```diff
@@ -4,7 +4,7 @@ Our development setup assumes a LINUX/BSD environment.
 
 ## Project Setup
 
-1. Install Node v20.
+1. Install Node v22 `nvm install lts/jod && nvm use lts/jod`
```

Comment:
```suggestion
1. Install Node v22 `nvm install && nvm use`
```

this will read what's in the nvmrc file

Reply:
> if i see "nvm use", i then have to go and check what is in the .nvmrc

why does the reader need to check what's in the `.nvmrc`? copy/pasting `nvm install && nvm use` is the same as copy/pasting `nvm install lts/jod && nvm use lts/jod` but less chance of outdated information since the `nvmrc` becomes the source of truth

Reply:
maybe I'm not understanding the reason but I don't agree with this argument

> There are many repositories out there whose set up instructions are simply "yarn <insert_script_name_here>"

why is this a bad thing? we have many commands in our package.json that simplify the workflow for the user. e.g. `yarn start`, `yarn lint`, `yarn test`, etc. should the user not trust these predefined configurations?

I guess I just don't understand the concern. for users that aren't familiar with nvm, adding `lts/jod` won't provide value for them. and for users that are familiar with nvm, if they want to know they node version, they'll look at the `nvmrc` file